src/index.js

```diff
@@ -4,3 +4,9 @@ import App from './components/app';
 import './css/global-styles';
 
 ReactDOM.render(<App />, document.getElementById('root'));
+
+window.addEventListener('beforeunload', function(e) {
```

Comment:
not 100% sure where this would ideally go! 😕

Reply:
I recommend we put it in Wayfinder. Also, are you sure this is only triggered when the user has already started the questionnaire? It doesn't seem to be conditional on anything

Reply:
That's right it isn't done on a condition, seems to be something with the browser. When playing around with it in my branch it won't ask "Are you sure?" unless I actually start answering the questions.

Reply:
As this works, I would leave where it is. It is also not React, so makes sense to keep it in index.html